java-checks/src/main/java/org/sonar/java/checks/FilesExistsJDK8Check.java

```diff
@@ -45,7 +45,7 @@
 
   @Override
   public boolean isCompatibleWithJavaVersion(JavaVersion version) {
-    return version.asInt() == 8;
+    return version.isJava8Compatible();
```

Comment:
Looking at the [description of the rule](https://jira.sonarsource.com/browse/RSPEC-3725), it seems that we suggest not using `File.exists` with Java 8 but `isJava8Compatible` checks if the version is greater or equal to 8.
So this change would alter the behavior of the rule and the analyzer.
Is it the change we expected to implement?

Reply:
Decided to leave the check as is, cause our compatibility message says that if not set, we assume 8 or greater, not just 8